jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -4596,12 +4601,27 @@ static void doInitCacheIds() {
 	constructorParameterTypesField.setAccessible(true);
 	methodParameterTypesField.setAccessible(true);
 	if (reflectCacheEnabled) {
+		J9VMInternals.prepare(Field.class);
+		J9VMInternals.prepare(Executable.class);
```

Comment:
Unless there's a reason for this order, please prepare classes in alphabetical order.